docs/agent-server-compatibility.asciidoc

```diff
@@ -1,11 +1,15 @@
 [[agent-server-compatibility]]
 === {apm-agent} compatibility
 
-The chart below outlines the compatibility between different versions of the APM agents and the APM integration/server.
+The chart below outlines the compatibility between different versions of Elastic APM agents and extensions with the APM integration.
 
 [options="header"]
 |====
-|Language |{apm-agent} version |APM integration/server version
+|Language |{apm-agent} version |APM integration version
+// APM AWS Lambda extension
+.1+|**APM AWS Lambda extension**
+|`1.x` |≥ `6.5`
```

Comment:
@kruskall Do you know what the minimum compatible APM integration/server version is for the APM AWS Lambda extension?

Reply:
no, I'll investigate and let you know

Reply:
👋 Any update on this? If we're not sure, I'd like to remove the update to the comparison table from this PR so we can merge the release note links.

Reply:
The extension will work with old versions of APM Server, but we only started adding the `faas.*` fields from 8.0.0+. Aside from metrics, they were all there in 8.2.0, so I'd suggest we set that as the minimum version. Full support for Lambda metrics will come in 8.4.0.

Reply:
```suggestion
|`1.x` |≥ `8.2`
```